Frontend/src/pages/marketplace/ProductDetail.jsx

```
@@ -173,7 +194,7 @@ const ProductDetail = () => {
   };
   
   const handleQuantityChange = (e) => {
-    setQuantity(parseInt(e.target.value) || 1);
+    setQuantity(Math.max(1, parseInt(e.target.value) || 1));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)